### PR TITLE
Clarify README.md content

### PR DESCRIPTION
**Pull request by shoffmeister**
While attempting to get JCasC up and running as a container, the relation between "the plugin manager", jenkins-plugin-cli and the exact behaviour exhibited were rather confusing.

This PR attempts to clarify some details which may deconfuse some readers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,28 +95,29 @@ plugins:
     source:
       version: 2.19-rc289.d09828a05a74
   ...
-tool:
-  ...
 ```
 
-Any root object other than `plugins` will be ignored by the plugin installation manager tool. As with the plugins.txt file, version and url are optional, and if no version is entered, the latest version is the default. If a groupId is entered, the tool will try to download the plugin from the incrementals repository.
+As with the plugins.txt file, version and URL are optional. If if no version is provided, the latest version is used by default. If a groupId is provided, the tool will try to download the plugin from the Jenkins incrementals repository.
```
Comment (oleg-nenashev) [inline README.md:100]: "If if"

**Review by oleg-nenashev — COMMENTED** [review]
Thanks! Just one typo, the rest looks good. While we are here, it might make sense to start  adding line breaks inside paragraphs so that it is easier to review and edit

File: README.md (hunk 1 above)
Comment (shoffmeister) [reply README.md:100]: Previously: ", and if ... . If".
Now: "If ... . If".

Granted, that won't make me a candidate for the Nobel Prize in Literature, but I tried to apply restraint in my surgery, not play whack-a-mole with a sledgehammer. :P

> Just one typo

My eyes are tired, sorry - I don't see it?

Comment (jetersen) [reply README.md:100]:
```suggestion
As with the plugins.txt file, version and URL are optional. If no version is provided, the latest version is used by default. If a groupId is provided, the tool will try to download the plugin from the Jenkins incrementals repository.
```

**Review by timja — APPROVED** [review]

**Pull request merged**